# limits of the map
# resolution of coordinates in meters
resolution: 0.3

# [xmin, xmax], [ymin, ymax]
bounds: [[0.0, 34.0], [0.0, 48.0]]

# list of all obstacles

# total number of edges of each obstacle
obs_total: 11
obs_total_vertices: 41

# coordinates of the vertices of each
obstacles: [[[12.0, 6.0], [14.5, 3.5], [17.0, 5.5], [17.0, 8.5], [14.0, 8.0]],
              [[24.0, 6.0], [26.0, 3.5], [31.0, 7.5], [24.5, 9.5]],
              [[34.0, 26.0], [10.0, 26.0], [10.0, 12.0], [34.0, 12.0]],
              [[0.0, 26.0], [0.0, 6.0], [4.0, 6.0], [4.0, 26.0]],
              [[4.0, 32.0], [6.0, 30.0], [8.0, 32.0]],
              [[17.0, 32.0], [18.0, 30.0], [19.0, 32.0]],
              [[0.0, 36.0], [0.0, 32.0], [29.0, 32.0], [29.0, 36.0]],
              [[34.0, 36.0], [33.0, 34.0], [34.0, 32.0]],
              [[6.0, 44.0], [2.0, 43.0], [2.0, 39.0], [6.0, 38.0], [8.0, 41.0]],
              [[11.0, 48.0], [17.0, 41.0], [14.0, 48.0]],
              [[30.0, 48.0], [22.0, 40.0], [32.0, 48.0]]]
